src/AutoRest.CSharp/Properties/launchSettings.json

```diff
@@ -2,487 +2,487 @@
   "profiles": {
     "Accessibility": {
       "commandName": "Project",
-      "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestProjects\\Accessibility\\Generated"
+      "commandLineArgs": "--standalone $(SolutionDir)/test/TestProjects/Accessibility/Generated"
```

Comment:
Will folks be able to debug the project locally on Windows machines with this change?

Reply:
Yes:
![image](https://user-images.githubusercontent.com/8933858/148609406-afc0981f-0969-4234-b2af-bb293f21e500.png)